class-tgm-plugin-activation.php

```diff
@@ -1337,6 +1362,22 @@ protected function display_settings_errors() {
 		}
 
 		/**
+		 * Display admin notice if plugins have been force activated.
+		 *
+		 * @since 2.6.0
+		 */
+		public function display_forced_activation_notice() {
+			if ( $force_activated_plugins = get_transient( 'tgmpa_force_activated_plugins' ) ) {
+				?>
```

Comment:
Re: my edited remark on line 456 - while unlikely, there may be a lingering transient which hasn't been displayed yet for a plugin which in the mean time has been deactived again ? Do we need to check if the force activated plugins we're showing the notice for are still activated before showing them in the notice ?